apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -4585,6 +4585,14 @@ label {
     color: var(--dark-blue01) !important;
 }
 
+.primary-color {
```

Comment:
Rohit why you added this, We have color.css file you can find colors there

Reply:
@shalinee-walkover there was existing one as you can see above of this css, I  change primary_clr to primary-color due PR review changes I can not find primary blue color that why have added this one